src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```
@@ -2220,3 +2220,9 @@ services:
     public: false
     arguments:
       $locales: "@=service('prestashop.adapter.legacy.context').getLanguages()"
+
+  PrestaShopBundle\Form\Admin\Type\ApeType:
+    calls:
```

I think we can set the service visibility public to false, no ?

```suggestion
  PrestaShopBundle\Form\Admin\Type\ApeType:
    public: false
    autowire: true
    calls:
```